utils/server/grpc.go

```diff
@@ -408,8 +409,13 @@ func (s *Server) RealTime(stream pb.SpaceCloud_RealTimeServer) error {
 				return true
 			}
 
+			options := new(realtime.LiveQueryOptions)
```

Comment:
Lets remove these steps complete. Lets send the empty feed data array as is. On the client we can ignore the message if feed data array is empty